src/Symfony/Component/Form/Extension/Core/Type/BirthdayType.php

```diff
@@ -35,4 +37,17 @@ public function getBlockPrefix(): string
     {
         return 'birthday';
     }
+
+    public function buildView(FormView $view, FormInterface $form, array $options): void
+    {
+        parent::buildView($view, $form, $options);
```

Comment:
we generally don't call the empty methods of the AbstractType

Reply:
Didn't see that it was empty sorry :(